scripts/ci/Dockerfile

```diff
@@ -4,13 +4,18 @@ ARG DOCKER_IMAGE_REGISTRY=public.ecr.aws
 FROM $DOCKER_IMAGE_REGISTRY/docker/library/node:18-alpine3.15 as deps
 
 RUN apk add -U git
+RUN apk add --no-cache aws-cli
 
 WORKDIR /build
 
 # Adding and installing packages
 ADD package.json yarn.lock .yarnrc.yml ./
 COPY .yarn/ ./.yarn
 RUN apk add --update --no-cache python3 build-base gcc && ln -sf /usr/bin/python3 /usr/bin/python
+
+RUN yarn config set npmRegistryServer "https://npm-upstream-821090935708.d.codeartifact.eu-west-1.amazonaws.com/npm/github-runner-npm-cache/"
+RUN yarn config set 'npmRegistries["https://npm-upstream-821090935708.d.codeartifact.eu-west-1.amazonaws.com/npm/github-runner-npm-cache/"].npmAuthToken' `aws codeartifact get-authorization-token --domain npm-upstream --domain-owner 821090935708 --region eu-west-1 --query authorizationToken --output text`
```

Comment:
should  we maybe get the token before building the image and pass it in as docker secret?

Reply:
Does that not leave a trace of the token in github CI uncovered?